src/test/java/hudson/tasks/MailerTest.java

```diff
@@ -37,12 +40,21 @@
 
 import jenkins.model.JenkinsLocationConfiguration;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertNull;
+
 
 /**
  * @author Kohsuke Kawaguchi
  */
-public class MailerTest extends HudsonTestCase {
+public class MailerTest {
+
+    @Rule
+    public JenkinsRule rule = new JenkinsRule();
+
     @Bug(1566)
```

Comment:
`@Bug(42)` should be converted to `@Issue("JENKINS-42")`, I do not see any need for additional marking.